README.md

```diff
@@ -9,24 +9,28 @@ relation.
 
 ```javascript
 function mystery(n) {
-    if(n <= 1)
-        return;
-    else {
-        mystery(n / 3);
-        var count = 0;
-        mystery(n / 3);
-        for(var i = 0; i < n*n; i++) {
-            for(var j = 0; j < n; j++) {
-                for(var k = 0; k < n*n; k++) {
-                    count = count + 1;
-                }
-            }
+  if (n <= 1) return;
+  else {
+    mystery(n / 3);
+    var count = 0;
+    mystery(n / 3);
+    for (var i = 0; i < n * n; i++) {
+      for (var j = 0; j < n; j++) {
+        for (var k = 0; k < n * n; k++) {
+          count = count + 1;
         }
-        mystery(n / 3);
+      }
     }
+    mystery(n / 3);
+  }
 }
 ```
 
 Add your answer to this markdown file. [This
 page](https://docs.github.com/en/get-started/writing-on-github/working-with-advanced-formatting/writing-mathematical-expressions)
 might help with the notation for mathematical expressions.
+I had ai help with understanding the master theorum.  
+
+Let T(n) represent the runtime.  The function consists of three recursive calls to mystery(n/3), contributing 3T(n/3), and a triple nested loop with bounds O(n^2), O(n), and O(n^2), resulting in O(n^5) work.  This gives the recurrence T(n) = 3T(n/3) + O(n^5).  Using the Master Theorem for divide and conquer recurrences, we calculated p = log(base three) 3 = 1.  Comparing p and d = 5, we find p < d, meaning the non recursive work O(n^5) dominated the runtime.  The tight big-O bound for the runtime of the function is O(n^5).
```

Comment:
You cannot have an asymptotic expression in a $T(n)$ expression. Please use the same method we used in class to solve the recurrence relation.